Assignment1/test-output/Default suite/Default test.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Generated by org.testng.reporters.JUnitXMLReporter -->
-<testsuite ignored="0" hostname="rizwans-mbp.mynet" failures="0" tests="7" name="Default test" time="5.357" errors="0" timestamp="17 Jul 2020 15:38:42 GMT">
-  <testcase classname="files.DelayedResponseTests" name="test_Response_Time_returns_HTTP_200_UStory15" time="4.288"/>
-  <testcase classname="files.DelayedResponseTests" name="test_get_delayed_response_returns_HTTP_200_UStory15" time="0.438"/>
-  <testcase classname="files.DelayedResponseTests" name="test_get_delayed_response_returns_HTTP_200_UStory15" time="0.111"/>
-  <testcase classname="files.DelayedResponseTests" name="test_get_delayed_response_returns_HTTP_200_UStory15" time="0.141"/>
+<testsuite ignored="0" hostname="rizwans-mbp.mynet" failures="0" tests="7" name="Default test" time="6.409" errors="0" timestamp="22 Jul 2020 11:36:05 GMT">
```

Comment:
Test output should not be a part of the repository. Please delete the whole `test-output` directory from the repository.

Reply:
removed